### PR TITLE
Create etherlite.json

### PR DESCRIPTION
**Pull request by etherlite-org**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
It's missing width and heitght

File: _data/icons/etherlite.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy",
+      "width": 80,
```
Comment (ligi) [inline _data/icons/etherlite.json:4]:
```suggestion
      "width": 88,
```

File: _data/icons/etherlite.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+      "height": 80,
```
Comment (ligi) [inline _data/icons/etherlite.json:5]:
```suggestion
      "height": 88,
```

**Review by ligi — CHANGES_REQUESTED** [review]
```
✓ ~ ipfs get QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy             14:46:32
Saving file(s) to QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy
 5.34 KiB / 5.34 KiB [==============================================] 100.00% 0s
✓ ~ identify QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy             14:46:36
QmbNAai1KnBnw4SPQKgrf6vBddifPCQTg2PePry1bmmZYy PNG 88x88 88x88+0+0 8-bit sRGB 5472B 0.000u 0:00.000
```

File: _data/icons/etherlite.json (hunk 2 above)
Comment (etherlite-org) [reply _data/icons/etherlite.json]: Updated the requested dimensions for logo.

**Pull request merged**